packages/web-components/src/components/ic-pagination/ic-pagination.tsx

```diff
@@ -84,6 +88,17 @@ export class Pagination {
     this.icPageChange.emit({ value: this.currentPage });
   }
 
+  @Method()
```

Comment:
Is this a public method or will it be internal? If it is public, there needs to be a description so it appears within the guidance site props table...